### DISCUSSION TITLE
Can't launch GUI on terminal

### DISCUSSION THREAD

**Discussion by DearNebReddey** [wehr-lab/autopilot#135] [Q&A] [answered]
Hi,

I followed the installation of autopilot everything was working perfectly, it has been a while since i did the installation  a month maybe but now when I launch this is what I get 

![Screenshot from 2021-10-15 11-06-51](https://user-images.githubusercontent.com/92102377/137463738-865fac36-dd98-4b5c-9df3-9d430bff615f.png)

please find attached screen shot about the terminal error message

and it is working fine for the setup "python3 -m autopilot.setup"

Regards

**Comment by DearNebReddey** [comment, marked as answer]
Fixed using this solution
[https://github.com/stepjam/PyRep/issues/76#issuecomment-559099448](url)

**Reply by sneakers-the-rat** [reply]
Glad you figured it out! Sorry I've been deep into a few other projects and haven't had time to keep an eye out here. I'll add this to the always-growing list of things to add to the docs and wiki :)
